db/migrate/20160523155343_add_enabled_column_to_service.rb

```diff
@@ -0,0 +1,5 @@
+class AddEnabledColumnToService < ActiveRecord::Migration
+  def change
+    add_column :services, :enabled, :boolean, default: false, null: false
```

Comment:
More of an open question - would you typically add an index to this kind of column? As it will be used fairly often...?